### ISSUE TITLE
(SP: 0.5) Console warning when closing Year filter popover

### ISSUE THREAD

**Issue by qqwz0** [ita-social-projects/Liatoshynsky#1082]
Console shows an accessibility warning when using the year filtering:
Blocked aria-hidden on an element because its descendant retained focus. The focus must not be hidden from assistive technology users. Avoid using aria-hidden on a focused element or its ancestor. Consider using the inert attribute instead, which will also prevent focus. For more details, see the aria-hidden section of the WAI-ARIA specification at https://w3c.github.io/aria/#aria-hidden.
Element with focus: <div.MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation8 MuiPopover-paper MuiMenu-paper css-h3sq9k-MuiPaper-root-MuiPopover-paper-MuiMenu-paper>
Ancestor with aria-hidden: <div.MuiPopover-root MuiMenu-root MuiModal-root css-pa188r-MuiModal-root-MuiPopover-root-MuiMenu-root> 

## Steps to Reproduce

1. Open year filtering and use the drag.
2. Click outside.
3. Check DevTools console.

## Expected Result

No accessibility warnings.
When the menu closes, focus is moved to a valid element (usually the trigger button) and no focused element is inside an aria-hidden subtree.
